ci/scripts/release.sh

```diff
@@ -54,6 +56,12 @@ if [[ -n "${BUILDKITE_TAG+x}" ]]; then
   echo "--- Release upload asset"
   tar -czvf risingwave-"${BUILDKITE_TAG}"-x86_64-unknown-linux.tar.gz risingwave
   gh release upload "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-x86_64-unknown-linux.tar.gz
+
+  echo "--- Release build and upload risingwave connector node jar asset"
+  git clone https://"$GITHUB_TOKEN"@github.com/risingwavelabs/risingwave-connector-node.git
+  cd risingwave-connector-node && git checkout ${CONNECTOR_NODE} && mvn -B package -Dmaven.test.skip=true
+  cd assembly/target && mv risingwave-connector-1.0.0.tar.gz risingwave-connector-"${BUILDKITE_TAG}".tar.gz
```

Comment:
The version is fixed at `1.0.0` ? 🤣

Reply:
😂

Reply:
Actually we can add an arg `-Drelease.version=0.1.16` to `mvn -B package -Dmaven.test.skip=true` to specify the output version of the tar file.